README.md

```diff
@@ -1,3 +1,171 @@
+
+## 🛠 Built With <a name="built-with">React</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">React.js</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[GitFlow]**
+- **[React Router]**
+- **[React Components]**
+- **[React JSX]**
+- **[React Redux]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo">Under construction</a>
+
+- [Live Demo Link](Under construction)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder
+  git clone https://github.com/Zafron047/Book_Store_React.git
+
+
+### Install
+
+Install this project with:
+
+```
+npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+-Run the index.html with your preferred browser.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Run Tests
+
+You can run tests to check for linter errors by running the following commands: <br/>
+
+ - for testing css errors
+```
+npx stylelint "**/*.{css,scss}"
+```
+ - for Javascript errors
+```
+npx eslint "**/*.{js,jsx}" 
+```
+
+### Deployment
+You can deploy this project using:
+- Free deployment services like GitHub pages.
+- Any deployment services of your choice.
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 MD. Nohain Islam
+- GitHub: [@githubhandle](https://github.com/Zafron047)
+- Twitter: [@twitterhandle](https://https://twitter.com/NohainZ)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nohain-islam-0261b4148/)
+<br><br>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- **[Improve the UI]**
+- **[Add Redux]**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Zafron047/Book_Store_React/issues).
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can recommend to your friend, family and colleagues who want to use my services
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+**I would like to thank microverse for guiding and providing with all great material to do my projects**
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
 # Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
- [ ] Kindly remove this line to the end of your readme document. It contains a lot of generic texts generated by React.

Reply:
When I was updating the readme file. I kept the react generated texts that were relevant to my project and removed the irrelevant ones. Do I still need to remove texts relevant to my project?